### PR TITLE
Creates the AWS Comprehend Target Documentation

### PR DESCRIPTION
**Pull request by JeffNeff**

### CODE REVIEW

File: docs/targets/awscomprehend.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Event Target for AWS Comprehend
+
+This event target receives [CloudEvents][ce] over HTTP and sends them to [AWS Comprehend](https://aws.amazon.com/comprehend/)
```
Comment (mattray) [inline docs/targets/awscomprehend.md:3]: is the CloudEvents link missing?

Comment (mattray) [reply docs/targets/awscomprehend.md:3]: does the footnote thing at the bottom work? I've never seen that before

File: docs/targets/awscomprehend.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+For more information about using AWS Comprehend, please refer to the [documentation][https://docs.aws.amazon.com/comprehend/].
+
+## Event Types
+### Arbitrary
+This target will consume arbitrary events and analyzes each of the key values sentiment. It then combines the scores and 
+returns the analysis in a response event of type `io.triggermesh.targets.aws.comprehend.result`
+
+
+[ce]: https://cloudevents.io/
+[ce-jsonformat]: https://github.com/cloudevents/spec/blob/v1.0/json-format.md
```
Comment (mattray) [inline docs/targets/awscomprehend.md:33]: if that footnote thing does work... the ce-jsonformat isn't used?

Comment (JeffNeff) [reply docs/targets/awscomprehend.md:33]: no that is copy pasta good catch

Comment (JeffNeff) [reply docs/targets/awscomprehend.md]: updated :)

**Review by mattray — APPROVED** [review]
:shipit:

**Pull request merged**